lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -46,39 +49,39 @@
 
             screenReaderTitle: gettext('Profile Image'),
 
-            imageUrl: function () {
+            imageUrl: function() {
                 return this.model.profileImageUrl();
             },
 
-            imageAltText: function () {
-                return interpolate_text(
-                    gettext("Profile image for {username}"), {username: this.model.get('username')}
+            imageAltText: function() {
+                return HtmlUtils.interpolateHtml(
```

Comment:
According to guidelines, this would still be plain text at this point and should use `StringUtils.interpolate()`.